demo-extended/src/main/java/com/novoda/simplechromecustomtabs/demo/ExtendedDemoActivity.java

```diff
@@ -42,6 +48,15 @@ public void onClick(View v) {
         }
     };
 
+    private final OnWebLinkClickedListener onWebLinkClickedListener = new OnWebLinkClickedListener() {
+        @Override
+        public void onClick(Uri uri) {
+            SimpleChromeCustomTabs.getInstance().withFallback(navigationFallback)
```

Comment:
This and the other can be extracted to a `navigateTo(Uri uri)` method I believe.